app-infrastructure/auth-hpds-instance.tf

```diff
@@ -22,7 +22,7 @@ data "template_cloudinit_config" "auth_hpds-user-data" {
 }
 
 resource "aws_instance" "auth-hpds-ec2" {
-  count = var.env_is_auth ? 1 : 0
+  count = var.include_auth_hpds ? 1 : 0
```

Comment:
do we need this variable? since auth will only exist in auth, we can do `!var.env_is_open_access ? 1 : 0`. saves us from defining one more variable

Reply:
I disagree. In the future we will like to include an "Auth" HPDS instance in open access. It will contain a different dataset then our current Auth environment. By using my approach we have the freedom to include it in the future.